src/main.py

```diff
@@ -50,6 +52,67 @@ def is_active_command(server_id: str, command_name: str) -> bool:
     return False
 
 
+def create_user(member: member_type) -> bool:
+    """Create a user in the firestore db."""
+    server_id = str(member.guild.id)
+    user_collection: CollectionReference = db.collection("servers").document(server_id).collection("users")
+    doc_ref: DocumentReference = user_collection.document(str(member.id))
+    doc: DocumentSnapshot = doc_ref.get()
+
+    # If user doesn't exist, create it with user_id as the document id
+    if not doc.exists:
+        doc_ref.set(
+            {
+                "avatar_url": str(member.avatar_url),
+                "exp_toward_next_level": 0,
+                "level": 1,
```

Comment:
default level is level 0

Reply:
```py
DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S"
# ...
{
                    "total_exp": 0,
                    "exp_toward_next_level": 0,
                    "level": 0,
                    "message_count": 0,
                    "last_message_timestamp": datetime.now().strftime(DATETIME_FORMAT),
                    "username": username,
                    "avatar_url": avatar_url,
                }
```
Add the last_message_timestamp as well please

Reply:
Done